README.md

```diff
@@ -18,7 +18,27 @@ NYC-DB builds a postgresql database containing the following datasets:
 - 2017 Marshal Evictions
 - ECB / Oath Hearings
 
-NYC-DB is a python3 command line program that downloads and loads datasets into postgres.
+NYC-DB includes a python3 command line program that downloads the data and loads datasets into postgres.
+
+While loading data into postgres, it also transforms the raw data in various ways: by cleaning up bad data, formatting it more consistently, creating indexes on frequently-used columns, and more.
+
+## Database structure
+
+The datasets in NYC-DB come from disparate sources, but almost all of them can be
```

Comment:
I'm not sure "disparate" is the right word here. They *all* essentially come from City and State government agencies.